### DISCUSSION TITLE
Only use glide.js carousel on mobile screen size

### DISCUSSION THREAD

**Discussion by ajithlalrg** [glidejs/glide#600] [General]
I am Using the Glide js library for the carousel, which I went through the document https://glidejs.com/ and it's working fine. But the scenario I am trying to achieve is the carousel should not work on desktop view and it should only work on mobile view.  I have tried the below example code

**Comment by MythicOrange** [comment]
there is no example code given.

also this should easily be doable with some css and a breakpoint in the glide's config.
